DominicanWhoCodes.Blazor/wwwroot/data/devs-pics/jose-segura.jpg

```diff
@@ -0,0 +1,9 @@
+<HTML><HEAD>
```

Comment:
I can't see your picture... check this out

Reply:
i fixed that